### ISSUE TITLE
hello

### ISSUE THREAD

**Issue by shamirahim111** [sixhobbits/scripts#1]
Hi.

saw your project while searching for phocos any-grid inverters.

apparently your software pulls data from phocos cloud. do you know how much the subscription cost is for the cloud service? i am interested in buying the connector but not sure if i want to pay a subscription

**Comment by sixhobbits** [comment]
no, it came free with the inverter for an 'initial period' but so far they have not asked me to pay. February will be two years so I'll see if it still works after that -- otherwise I think you can plug in something more hackable (I came across https://www.victronenergy.com/communication-centres/globallink-520 before but haven't looked too much into different options yet)
